src/main/java/org/demo/service/cxbox/inner/MeetingReadService.java

```diff
@@ -41,15 +45,11 @@ public class MeetingReadService extends VersionAwareResponseService<MeetingDTO,
 
 	private static final String MESSAGE_TEMPLATE = "Status: %s; \nMeeting Result: %s";
 
-	public MeetingReadService(MeetingRepository meetingRepository, UserRepository userRepository,
-			SessionService sessionService,
-			MeetingStatusModelActionProvider statusModelActionProvider, MailSendingService mailSendingService) {
-		super(MeetingDTO.class, Meeting.class, null, MeetingReadMeta.class);
-		this.meetingRepository = meetingRepository;
-		this.userRepository = userRepository;
-		this.sessionService = sessionService;
-		this.statusModelActionProvider = statusModelActionProvider;
-		this.mailSendingService = mailSendingService;
+	@Getter
+	private final Class<MeetingReadMeta> fieldMetaBuilder = MeetingReadMeta.class;
+
+	public final Class<? extends FieldMetaBuilder<MeetingDTO>> getFieldMetaBuilder() {
```

Comment:
this duplicates "private final Class<MeetingReadMeta> fieldMetaBuilder = MeetingReadMeta.class;", no?